tasks/shell_helpers/tldr.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: Clone tldr-sh-client
+  git:
+    repo: https://github.com/raylee/tldr-sh-client.git
+    dest: ~/projects/shell-helpers/tldr-sh-client
+    depth: "1"
```

Comment:
Could you add `version` to this with `local_system_tldr` and add it additionally to `defaults/main/versions.yml`.
Even though it's just `master` it helps to add some control flow.